mantidproject/mantid
# Indirect - Data Analysis Interfaces - Decrease Size of Difference Preview Plot

Recently, the difference between the sample and the fit in the Indirect Data Analysis tabs was moved to a separate preview plot, below the existing one and of the same size.

This PR halves the size of the preview plot and ensures the plots within the Data Analysis tabs with only a single preview (non-fitting tabs) do not have stretched plots.

**To test:**
1. Navigate to Indirect > Interfaces > Data Analysis.
2. Navigate through each Data Analysis Tab, ensuring where there is a single plot, that it is not stretched across the entire height of the interface. Where there are two plots, ensure the second plot is around half the size of the first.



Fixes #21121. 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
